python-frontend/typeshed_serializer/resources/custom/aws_cdk/aws_ec2/__init__.pyi

```diff
@@ -25,3 +25,8 @@ class LaunchTemplate(CustomStubBase):
 class SecurityGroup(CustomStubBase):
```

Comment:
Did you forget to commit updated protobuf file?

Reply:
I guess so :/ I took care of not comitting the third_party_libraries file but apparently also didn't commit the custom ones.
Sorry for that, incoming in the next commit.